### ISSUE TITLE
Question: Can multiple segment IDs be assigned to the same voxel?

### ISSUE THREAD

**Issue by kabilar** [scalableminds/webknossos#8032]
## Detailed Description
Within a single segmentation layer (i.e. volume annotation), can multiple segment IDs be assigned to the same voxel?  Perhaps if the segment IDs are in different groups (for example see image below)?  I just wanted to confirm as I have tried this on the application but it doesn't seem to be the case.

![image](https://github.com/user-attachments/assets/4eaad0d5-2aa1-4d44-b533-eaf46e4631cc)

## Context
We often have overlapping segmentations, so we want to be able to address this use case.  Since multiple segmentation layers cannot be displayed simultaneously (#5695), we were hoping to use segment IDs within a single segmentation layer to view multiple segmentations simultaneously but we also need some voxels to be assigned to multiple segmentations.

Thank you.

cc @aaronkanzer @jingjingwu1225 @chiaramff @balbasty

**Comment by philippotto** [comment]
> Within a single segmentation layer (i.e. volume annotation), can multiple segment IDs be assigned to the same voxel?

No, this is not possible. For each voxel in one segmentation layer, only one value can be stored. You need multiple segmentation layers if you need an overlap. Alternatively, you could express two overlapping segments (e.g., 1 and 2) with a special ID for the overlapping ids (e.g., 3), but there is no built-in support for handling this in a user-friendly manner.

I'll close this, because we don't plan to support this.

**Comment by kabilar** [comment]
Thank you, @philippotto.
